HW_4/hw_task_2.py

```diff
@@ -0,0 +1,25 @@
+# Напишите функцию принимающую на вход только ключевые
+# параметры и возвращающую словарь, где ключ — значение
+# переданного аргумента, а значение — имя аргумента. Если
+# ключ не хешируем, используйте его строковое представление.
+
+"""Не могу понять, где ошибка. Говорит, что это позиционные аргументы,
+но ведь задано **kwargs, а это ключевые"""
+
+
+def func(**kwargs):
+    dict = {}
+    for key, value in kwargs.items():
+        if value.__hash__() is None:
+            dict[key] = [str(value)]
+        else:
+            dict[key] = value
+    return dict
+
+
+key_1 = 8
+key_2 = "Python"
+key_3 = 3.14
+key_4 = [1, 2, 3]
+
+print(func(key_1, key_2, key_3, key_4))
```

Comment:
Нужно было сделать так:
```suggestion
print(func(key_1=8, key_2="Python", key_3=3.14, key_4=[1, 2, 3]))
```